### ISSUE TITLE
link to distant resources

### ISSUE THREAD

**Issue by sylvainlb** [read-write-web/rww-play#140]
How can we include in a container resources laying in a container on another server?
It would be great if one was allowed to use some sort of include to indicate that some container includes its own resources, and some distant resources in addition.

**Comment by bblfish** [comment]
So one way is to allow the container to be edited, so that any relation can be added there. Of course there you could not use the `ldp:contains` relation because that implies that when you delete the object the relation disappears from the container, which you cannot guarantee if your resource is on another server. You would then be able to add `<> rdfs:member <htttp://other.com/thing>`  to your container.

If is possible to add that to the server. I wonder if there is a way of doing what you want without this requirement. Do you have a use case?

**Comment by sylvainlb** [comment]
Yes, typically, I want a create a todo list, but some of my todos are on server A, and some on server B. It would be nice to point my script on a LDPC on A, and to get A to serve the whole list, or at least a notification that the client should look on B.
